isnts/views/auth.py

```diff
@@ -180,12 +180,22 @@ def render_form():
                 request.POST, emp_types=e_types)
 
             if employee_registration_form.is_valid():
+                data = employee_registration_form.cleaned_data
+                employee_secret_key = data['secret_key']
                 user = employee_registration_form.save()
                 user.set_password(user.password)
+                user.is_active = False
                 g = Group.objects.get(
                     id=employee_registration_form.cleaned_data['employee_type'])
                 g.user_set.add(user)
                 user.save()
+                nts_list = NTS.objects.all()
```

Comment:
toto sa da robit krajsie a setrit aj pamat aj rychlost, pozri si query funkciu `Model.objects.filter(parameter='value')`. Aby si to nemusel nacitavat cele a prechadzat to v cykle